api/types.go

```diff
@@ -561,9 +563,24 @@ const (
 // CdagType pin
 type PinType int
 
+func (pT *PinType) Parse(str string) {
```

Comment:
I don't know which one is better, but we have `TrackerStatusFromString(str string) TrackerStatus`, then for consisntency this should be `PinTypeFromString(...)`

Reply:
Im cool with that 😎